### DISCUSSION TITLE
32bit Custom Windows Client?

### DISCUSSION THREAD

**Discussion by Trouble123** [rustdesk/rustdesk#8278] [General]
Hi
Is there a way to generate a custom 32bit client? I need the custom client as i need to pass the PRO option of preset-address-book-name=XXXXX.

Or is there a way of doing this with the normal client? I tried renaming the installer exe to include preset-address-book-name=XXXX.exe, but that didnt work.

Thanks

**Comment by rustdesk** [comment]
Not yet for the time being
